README.md

```diff
@@ -1,21 +1,13 @@
 # Gravitas
 
-**TODO: Add description**
+An agent to control your infrastructure.
 
-## Installation
+## Values
 
-If [available in Hex](https://hex.pm/docs/publish), the package can be installed
-by adding `gravitas` to your list of dependencies in `mix.exs`:
-
-```elixir
-def deps do
-  [
-    {:gravitas, "~> 0.1.0"}
-  ]
-end
-```
-
-Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_doc)
-and published on [HexDocs](https://hexdocs.pm). Once published, the docs can
-be found at [https://hexdocs.pm/gravitas](https://hexdocs.pm/gravitas).
+The values that guide the design of Gravitas are:
 
+- *Observability*: You should be able to understand what Gravitas is doing easily. Users should be able to maintain an understanding of the problem from Gravitas' perspective. The user should be able to have a representation of the problem state shared with Gravitas and a representation of both the activities of Gravitas and other users using it. Users need to be able to see what Gravitas is doing and what it will do next relative to its state. And this without need for overly taxing cognitive activity
+- *Directability*: You need to be able to control and steer the direction Gravitas is going into when you recognize a need to intervene. And this without having to take full manual control. Gravitas should be flexible and able to take directions in the areas that needs supervision. And doing so while keeping the rest of its functionality active in an adapted way. As above, that need to be possible without overly taxing cognitive activity from the user.
+- *Learnability*: Gravitas should provide ways for the user to grow its expertise with it. Gravitas handle a complex problem state and is itself a complex system. We consider that moving our user from the first discovery of the system to being an expert in it is essential. That may be through experimentations in a simulated environment, documentations, talking with other experts, etc. This should be an integral part of the user experience of Gravitas.
+- *Safety*: Gravitas need to fill the highest profesional standard for safety, security and ethics. We recognise our place in the ecosystem of software. As much as possible, we provide fail-safe situations and sane defaults. We aim to make the safe path easy. If it has to choose between ressources and safety, Gravitas should balance its choice toward safety.
```

Comment:
Drop the "professional" from "highest professional standards" imho. It has been said too much in snake-oil context, so in my view it actually devalues the standards we strive towards.